Pylint: R1705

```bash
mmpose/apis/test.py:123:4: R1705: Unnecessary "else" after "return" (no-else-return)
mmpose/core/optimizer/builder.py:45:4: R1705: Unnecessary "else" after "return" (no-else-return)
mmpose/core/evaluation/eval_hooks.py:123:8: R1705: Unnecessary "else" after "return" (no-else-return)
mmpose/core/fp16/utils.py:18:4: R1705: Unnecessary "elif" after "return" (no-else-return)
mmpose/datasets/datasets/bottom_up/bottom_up_base_dataset.py:81:8: R1705: Unnecessary "else" after "return" (no-else-return)
mmpose/models/builder.py:21:4: R1705: Unnecessary "else" after "return" (no-else-return)
mmpose/models/losses/multi_loss_factory.py:92:8: R1705: Unnecessary "elif" after "return" (no-else-return)
mmpose/models/losses/mesh_loss.py:110:8: R1705: Unnecessary "else" after "return" (no-else-return)
mmpose/models/detectors/bottom_up.py:112:8: R1705: Unnecessary "else" after "return" (no-else-return)
mmpose/models/detectors/mesh.py:295:8: R1705: Unnecessary "else" after "return" (no-else-return)
mmpose/models/detectors/top_down.py:105:8: R1705: Unnecessary "else" after "return" (no-else-return)
mmpose/models/backbones/shufflenet_v1.py:311:8: R1705: Unnecessary "else" after "return" (no-else-return)
mmpose/models/backbones/regnet.py:309:8: R1705: Unnecessary "else" after "return" (no-else-return)
mmpose/models/backbones/resnet.py:664:8: R1705: Unnecessary "else" after "return" (no-else-return)
mmpose/models/backbones/mobilenet_v3.py:164:8: R1705: Unnecessary "else" after "return" (no-else-return)
mmpose/models/backbones/mobilenet_v2.py:83:12: R1705: Unnecessary "else" after "return" (no-else-return)
mmpose/models/backbones/mobilenet_v2.py:248:8: R1705: Unnecessary "else" after "return" (no-else-return)
mmpose/models/backbones/shufflenet_v2.py:284:8: R1705: Unnecessary "else" after "return" (no-else-return)
mmpose/models/backbones/utils/inverted_residual.py:107:12: R1705: Unnecessary "else" after "return" (no-else-return)

```

--- a/mmpose/apis/test.py
+++ b/mmpose/apis/test.py
@@ -122,21 +122,21 @@ def collect_results_cpu(result_part, size, tmpdir=None):
     # collect all parts
     if rank != 0:
         return None
-    else:
-        # load results of all parts from tmp dir
-        part_list = []
-        for i in range(world_size):
-            part_file = osp.join(tmpdir, f'part_{i}.pkl')
-            part_list.append(mmcv.load(part_file))
-        # sort the results
-        ordered_results = []
-        for res in zip(*part_list):
-            ordered_results.extend(list(res))
-        # the dataloader may pad some samples
-        ordered_results = ordered_results[:size]
-        # remove tmp dir
-        shutil.rmtree(tmpdir)
-        return ordered_results
+
+    # load results of all parts from tmp dir
+    part_list = []
+    for i in range(world_size):
+        part_file = osp.join(tmpdir, f'part_{i}.pkl')
+        part_list.append(mmcv.load(part_file))
+    # sort the results
+    ordered_results = []
+    for res in zip(*part_list):
+        ordered_results.extend(list(res))
+    # the dataloader may pad some samples
+    ordered_results = ordered_results[:size]
+    # remove tmp dir
+    shutil.rmtree(tmpdir)
+    return ordered_results
 
 
 def collect_results_gpu(result_part, size):

--- a/mmpose/core/evaluation/eval_hooks.py
+++ b/mmpose/core/evaluation/eval_hooks.py
@@ -122,8 +122,8 @@ class EvalHook(Hook):
         runner.log_buffer.ready = True
         if self.key_indicator is not None:
             return eval_res[self.key_indicator]
-        else:
-            return None
+
+        return None
 
 
 class DistEvalHook(EvalHook):

--- a/mmpose/core/fp16/utils.py
+++ b/mmpose/core/fp16/utils.py
@@ -29,5 +29,5 @@ def cast_tensor_type(inputs, src_type, dst_type):
     elif isinstance(inputs, abc.Iterable):
         return type(inputs)(
             cast_tensor_type(item, src_type, dst_type) for item in inputs)
-    else:
-        return inputs
+
+    return inputs

--- a/mmpose/core/optimizer/builder.py
+++ b/mmpose/core/optimizer/builder.py
@@ -48,5 +48,5 @@ def build_optimizers(model, cfgs):
             module = getattr(model, key)
             optimizers[key] = build_optimizer(module, cfg_)
         return optimizers
-    else:
-        return build_optimizer(model, cfgs)
+
+    return build_optimizer(model, cfgs)

--- a/mmpose/datasets/datasets/bottom_up/bottom_up_base_dataset.py
+++ b/mmpose/datasets/datasets/bottom_up/bottom_up_base_dataset.py
@@ -80,5 +80,5 @@ class BottomUpBaseDataset(Dataset):
         """Get the sample for either training or testing given index."""
         if self.test_mode:
             return self.prepare_test_img(idx)
-        else:
-            return self.prepare_train_img(idx)
+
+        return self.prepare_train_img(idx)

--- a/mmpose/models/backbones/mobilenet_v2.py
+++ b/mmpose/models/backbones/mobilenet_v2.py
@@ -82,8 +82,7 @@ class InvertedResidual(nn.Module):
         def _inner_forward(x):
             if self.use_res_connect:
                 return x + self.conv(x)
-            else:
-                return self.conv(x)
+            return self.conv(x)
 
         if self.with_cp and x.requires_grad:
             out = cp.checkpoint(_inner_forward, x)
@@ -247,8 +246,7 @@ class MobileNetV2(BaseBackbone):
 
         if len(outs) == 1:
             return outs[0]
-        else:
-            return tuple(outs)
+        return tuple(outs)
 
     def _freeze_stages(self):
         if self.frozen_stages >= 0:

--- a/mmpose/models/backbones/mobilenet_v3.py
+++ b/mmpose/models/backbones/mobilenet_v3.py
@@ -163,8 +163,7 @@ class MobileNetV3(BaseBackbone):
 
         if len(outs) == 1:
             return outs[0]
-        else:
-            return tuple(outs)
+        return tuple(outs)
 
     def _freeze_stages(self):
         if self.frozen_stages >= 0:

--- a/mmpose/models/backbones/regnet.py
+++ b/mmpose/models/backbones/regnet.py
@@ -308,5 +308,4 @@ class RegNet(ResNet):
 
         if len(outs) == 1:
             return outs[0]
-        else:
-            return tuple(outs)
+        return tuple(outs)

--- a/mmpose/models/backbones/resnet.py
+++ b/mmpose/models/backbones/resnet.py
@@ -663,8 +663,7 @@ class ResNet(BaseBackbone):
                 outs.append(x)
         if len(outs) == 1:
             return outs[0]
-        else:
-            return tuple(outs)
+        return tuple(outs)
 
     def train(self, mode=True):
         """Convert the model into training mode."""

--- a/mmpose/models/backbones/shufflenet_v1.py
+++ b/mmpose/models/backbones/shufflenet_v1.py
@@ -310,8 +310,7 @@ class ShuffleNetV1(BaseBackbone):
 
         if len(outs) == 1:
             return outs[0]
-        else:
-            return tuple(outs)
+        return tuple(outs)
 
     def train(self, mode=True):
         super().train(mode)

--- a/mmpose/models/backbones/shufflenet_v2.py
+++ b/mmpose/models/backbones/shufflenet_v2.py
@@ -283,8 +283,7 @@ class ShuffleNetV2(BaseBackbone):
 
         if len(outs) == 1:
             return outs[0]
-        else:
-            return tuple(outs)
+        return tuple(outs)
 
     def train(self, mode=True):
         super().train(mode)

--- a/mmpose/models/backbones/utils/inverted_residual.py
+++ b/mmpose/models/backbones/utils/inverted_residual.py
@@ -106,8 +106,7 @@ class InvertedResidual(nn.Module):
 
             if self.with_res_shortcut:
                 return x + out
-            else:
-                return out
+            return out
 
         if self.with_cp and x.requires_grad:
             out = cp.checkpoint(_inner_forward, x)

--- a/mmpose/models/builder.py
+++ b/mmpose/models/builder.py
@@ -23,8 +23,8 @@ def build(cfg, registry, default_args=None):
             build_from_cfg(cfg_, registry, default_args) for cfg_ in cfg
         ]
         return nn.Sequential(*modules)
-    else:
-        return build_from_cfg(cfg, registry, default_args)
+
+    return build_from_cfg(cfg, registry, default_args)
 
 
 def build_backbone(cfg):

--- a/mmpose/models/detectors/bottom_up.py
+++ b/mmpose/models/detectors/bottom_up.py
@@ -112,9 +112,8 @@ class BottomUp(BasePose):
         if return_loss:
             return self.forward_train(img, targets, masks, joints, img_metas,
                                       **kwargs)
-        else:
-            return self.forward_test(
-                img, img_metas, return_heatmap=return_heatmap, **kwargs)
+        return self.forward_test(
+            img, img_metas, return_heatmap=return_heatmap, **kwargs)
 
     def forward_train(self, img, targets, masks, joints, img_metas, **kwargs):
         """Forward the bottom-up model and calculate the loss.

--- a/mmpose/models/detectors/mesh.py
+++ b/mmpose/models/detectors/mesh.py
@@ -294,5 +294,4 @@ class ParametricMesh(BasePose):
 
         if return_loss:
             return self.forward_train(img, img_metas, **kwargs)
-        else:
-            return self.forward_test(img, img_metas, **kwargs)
+        return self.forward_test(img, img_metas, **kwargs)

--- a/mmpose/models/detectors/top_down.py
+++ b/mmpose/models/detectors/top_down.py
@@ -105,9 +105,8 @@ class TopDown(BasePose):
         if return_loss:
             return self.forward_train(img, target, target_weight, img_metas,
                                       **kwargs)
-        else:
-            return self.forward_test(
-                img, img_metas, return_heatmap=return_heatmap, **kwargs)
+        return self.forward_test(
+            img, img_metas, return_heatmap=return_heatmap, **kwargs)
 
     def forward_train(self, img, target, target_weight, img_metas, **kwargs):
         """Defines the computation performed at every call when training."""

--- a/mmpose/models/losses/mesh_loss.py
+++ b/mmpose/models/losses/mesh_loss.py
@@ -116,8 +116,7 @@ class MeshLoss(nn.Module):
             return (
                 conf *
                 self.criterion_joints_3d(pred_joints_3d, gt_joints_3d)).mean()
-        else:
-            return pred_joints_3d.sum() * 0
+        return pred_joints_3d.sum() * 0
 
     def vertex_loss(self, pred_vertices, gt_vertices, has_smpl):
         """Compute 3D vertex loss for the examples that 3D human mesh

--- a/mmpose/models/losses/multi_loss_factory.py
+++ b/mmpose/models/losses/multi_loss_factory.py
@@ -93,8 +93,7 @@ class AELoss(nn.Module):
                         torch.zeros(1).float(), device=pred_tag.device))
         elif num_tags == 1:
             return (_make_input(
-                torch.zeros(1).float(),
-                device=pred_tag.device), pull / (num_tags))
+                torch.zeros(1).float(), device=pred_tag.device), pull)
 
         tags = torch.stack(tags)
 

